charts/gateway-operator/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.5.6
+
+### Added
+
+- Update `ClusterRole` for 1.6.
```

Comment:
This is a bit confusing as the content is exactly the same as what in 0.5.5. Could tell what contents are changes in `ClusterRoles` here?

Reply:
72e0ebbd6ac159664b611519e147a667afa06a4c should make it a bit more clear. PTAL.